docs/administration/garbage-collection/_index.md

```diff
@@ -13,6 +13,7 @@ When you delete images from Harbor, space is not automatically freed up. You mus
 
     ![Garbage collection](../../img/garbage-collection.png)
 
+1. To delete untagged artifacts, select the check box **Delete Untagged Artifacts**. It's optional, the untagged artifacts will be GCed if selected. 
```

Comment:
please get rid of 'It's optional, the untagged artifacts will be GCed if selected.'